src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
do not forget to add more fallback fonts
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/cff5cecd-7624-4602-8a08-b09639863e8d)